src/components/UI/navbar/navbar.module.scss

```
@@ -138,6 +138,12 @@
 .homes {
   display: none;
 }
+.navBar li.navBarLoginLi .d-none {
+  display: none;
+}
+.navBar li.navBarLoginLi .d-none {
+  display: none;
+}
```

```suggestion
.navBar li.navBarLoginLi .d-none {
  display: none;
}
.navBar li.navBarLoginLi .d-none {
  display: none;
}
```
Do we need this, since we are not adding any class